docker-compose.yaml

```diff
@@ -42,17 +48,12 @@ services:
     restart: always
     ports:
       - "8081:8081"
-      - "8900:8900"
-      - "8901:8901"
-      - "8979:8979"
       - "10250:10250"
       - "10251:10251"
       - "10252:10252"
       - "10253:10253"
       - "10254:10254"
       - "10255:10255"
```

Comment:
Shouldn't have any bound ports or volumes outside the `override` file.  Otherwise they'll be bound to the CI build agent too which means concurrent builds will fail